.buildkite/shared-pipeline-vars

```diff
@@ -1 +1 @@
-export CI_TOOLKIT="automattic/a8c-ci-toolkit#3.7.0"
+export CI_TOOLKIT="automattic/a8c-ci-toolkit#3.7.1"
```

Comment:
Suggestion: we've been adopting this standard to separate the version numbers out of the full plugin name.

```suggestion
CI_TOOLKIT_PLUGIN_VERSION="3.7.1"

export CI_TOOLKIT="automattic/a8c-ci-toolkit#$CI_TOOLKIT_PLUGIN_VERSION"
```